types/pdfmake/interfaces.d.ts

```diff
@@ -453,7 +453,7 @@ export interface ContextPageSize {
 export interface BufferOptions {
     fontLayoutCache?: boolean;
     bufferPages?: boolean;
-    tableLayouts?: TableLayout;
+    tableLayouts?: {[key: string]: TableLayout};
```

Comment:
I believe this should be CustomTableLayout instead of TableLayout

Reply:
@jeralm you're right, I'll update this.